numpy/core/src/umath/ufunc_object.c

```diff
@@ -565,11 +565,12 @@ get_ufunc_arguments(PyUFuncObject *ufunc,
                     NPY_ORDER *out_order,
                     NPY_CASTING *out_casting,
                     PyObject **out_extobj,
-                    PyObject **out_typetup,
-                    int *out_subok,
-                    PyArrayObject **out_wheremask,
-                    PyObject **out_axes,
-                    int *out_keepdims)
+                    PyObject **out_typetup,  /* tuple of dtype */
```

Comment:
nit: `type: Tuple[np.dtype]` would match the mypy-style annotation used below

Reply:
done.